### PR TITLE
FEATURE - Add insight stacking plugin

### PR DESCRIPTION
**Pull request by hackerstanislav**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gauge_chart_plugin/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ naming convention with a Gauge chart component provided by the plugin.
 
 ![Gauge chart plugin](assets/Gauge_chart_plugin.png)
 
-## How to work with replace-by-tag plugin
+## How to work with gauge chart plugin
```
Comment (plavemar) [inline gauge_chart_plugin/README.md:8]: Why is here gauge chart, while this plugin is about insight stacking?

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR22

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-fossa-scanning-zuul-el7/21/ : SUCCESS in 6m 18s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-sonarqube-scan-zuul-gate/21/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-merge-compliance-check/22/ : SUCCESS in 29s

**Pull request merged**